bundle-workflow/tests/tests_test_workflow/test_run_bwc_test.py

```diff
@@ -0,0 +1,34 @@
+import os
+import unittest
+from unittest.mock import patch
+
+from run_bwc_test import BundleManifest, main
+
+
+class TestRunBwcTest(unittest.TestCase):
+    def setUp(self):
+        os.chdir(os.path.dirname(__file__))
```

Comment:
This is a side effect that can affect other tests, do what you need to do to avoid relying on current directory.